aider/README.md

```diff
@@ -0,0 +1,320 @@
+
+## Module Parameters
+
+| Parameter                        | Description                                                               | Type     | Default             |
+| -------------------------------- | ------------------------------------------------------------------------- | -------- | ------------------- |
+| `agent_id`                       | The ID of a Coder agent (required)                                        | `string` | -                   |
+| `folder`                         | The folder to run Aider in                                                | `string` | `/home/coder`       |
+| `install_aider`                  | Whether to install Aider                                                  | `bool`   | `true`              |
+| `aider_version`                  | The version of Aider to install                                           | `string` | `"latest"`          |
+| `use_screen`                     | Whether to use screen for running Aider in the background                 | `bool`   | `false`             |
+| `use_tmux`                       | Whether to use tmux instead of screen for running Aider in the background | `bool`   | `false`             |
+| `session_name`                   | Name for the persistent session (screen or tmux)                          | `string` | `"aider"`           |
+| `order`                          | Position of the app in the UI presentation                                | `number` | `null`              |
+| `icon`                           | The icon to use for the app                                               | `string` | `"/icon/aider.svg"` |
+| `experiment_report_tasks`        | Whether to enable task reporting                                          | `bool`   | `false`             |
+| `experiment_pre_install_script`  | Custom script to run before installing Aider                              | `string` | `null`              |
+| `experiment_post_install_script` | Custom script to run after installing Aider                               | `string` | `null`              |
+
+## Usage Examples
+
+### Basic setup
+
+```tf
+module "aider" {
+  count         = data.coder_workspace.me.start_count
+  source        = "registry.coder.com/modules/aider/coder"
+  version       = "1.0.0"
+  agent_id      = coder_agent.main.id
+  folder        = "/home/coder"
+  install_aider = true
+  aider_version = "latest"
+}
+```
+
+### With API key via environment variables
+
+```tf
+variable "anthropic_api_key" {
+  type        = string
+  description = "Anthropic API key"
+  sensitive   = true
+}
+
+variable "anthropic_model" {
+  type        = string
+  description = "Anthropic Model"
+  default     = "sonnet"
+}
+
+resource "coder_agent" "main" {
+  # ...
+}
+
+# Set API key and model using coder_env resource
+resource "coder_env" "anthropic" {
+  agent_id = coder_agent.main.id
+  name     = "ANTHROPIC_API_KEY"
+  value    = var.anthropic_api_key
+  secret   = true
```

Comment:
```suggestion
```
[`coder_env`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/env) does not have an attribute `secret`